distribution/std-lib/Table/src/Data/Order_Rule.enso

```diff
@@ -0,0 +1,24 @@
+from Base import all
+
+type Order_Rule
+    ## A rule used for sorting table-like structures.
+
+       Arguments:
+         - column: a value representing the underlying storage this rule is
+                   sorting by. This type does not specify the underlying
+                   representation of a column, assuming that the sorting engine
+                   defines its own column representation.
+         - comparator: a function taking two elements of the underlying column
+                       and returning an `Ordering`. The function may be
+                       `Nothing`, in which case a natural ordering will be used.
+                       Note that certain table backends (such us database
+                       connectors) may choose to ignore this field.
+         - order: specifies whether the table should be sorted in an ascending
+                  or descending order. The default value of `Nothing` delegates
+                  the decision to the sorting function.
```

Comment:
This should say something about it being the `Ordering` type in the standard library.